lib/snippets/kubernetes-deploy

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ruby
+
+# Usage: kubernetes-deploy <app's namespace> <kube context>
+
+# Prerequisites:
+#  - kubectl 1.5.1+ binary must be available in the shipit machine's path
+#  - ENV['KUBECONFIG'] must point to a valid kubeconfig file that includes all the contexts you want to deploy to
```

Comment:
Well, a service account would be more like a "user" in a kubeconfig, which can have multiple users. For example, we could have a user that is gcp auth'd, and one that is aws auth'd.